addons/web/static/src/views/kanban/kanban_column_quick_create.js

```diff
@@ -47,6 +47,12 @@ export class KanbanColumnQuickCreate extends Component {
 
         // Key Navigation
         useHotkey("escape", () => this.fold());
+        onPatched(() => {
+            if (this.unfolded) {
+                this.root.el.scrollIntoView({ behavior: "smooth" });
+            }
+            delete this.unfolded;
```

Comment:
```js
            if (this.unfolded && !this.props.folded) {
                this.root.el.scrollIntoView({ behavior: "smooth" });
                delete this.unfolded;
            }
```
because you can't know for sure that the patch is related to the unfold action (might be related to somethign that happened before)